### PR TITLE
[14.0][MIG] hotel_housekeeping: Migrated  module.

### PR DESCRIPTION
**Pull request by maitripatel19**
[14.0][MIG] hotel_housekeeping: Migrated  module.

### CODE REVIEW

File: hotel_housekeeping/report/activity_detail.rml [hunk 1]
```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0"?>
+<document filename="test.pdf">
+  <template pageSize="(595.0,842.0)" title="Test" author="Martin Simon" allowSplitting="20">
```
Comment (Trivedi-Vacha-SerpentCS) [inline hotel_housekeeping/report/activity_detail.rml]: why rml file added??

**Review by Trivedi-Vacha-SerpentCS — CHANGES_REQUESTED** [review]

**Comment by vimalpatelserpentcs** [comment]
@maitripatel19 @Trivedi-Vacha-SerpentCS , Travis is failed, I think you missed installing pre-commit. Can you please check that? Thank you.

File: hotel_housekeeping/README.rst [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fvertical--hotel-lightgray.png?logo=github
+    :target: https://github.com/OCA/vertical-hotel/tree/13.0/hotel_housekeeping
```
Comment (nikul-serpentcs) [inline hotel_housekeeping/README.rst]:
```suggestion
    :target: https://github.com/OCA/vertical-hotel/tree/14.0/hotel_housekeeping
```

File: hotel_housekeeping/README.rst [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    :alt: OCA/vertical-hotel
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/vertical-hotel-13-0/vertical-hotel-13-0-hotel_housekeeping
```
Comment (nikul-serpentcs) [inline hotel_housekeeping/README.rst]:
```suggestion
    :target: https://translation.odoo-community.org/projects/vertical-hotel-14-0/vertical-hotel-14-0-hotel_housekeeping
```

File: hotel_housekeeping/README.rst [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+Do not contact contributors directly about support or help with technical issues.
+
+Credits
+=======
+
+Authors
+~~~~~~~
+
+* Serpent Consulting                Services Pvt. Ltd.
```
Comment (nikul-serpentcs) [inline hotel_housekeeping/README.rst:76]:
```suggestion
* Serpent Consulting Services Pvt. Ltd.
```

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
nitpicking

File: hotel_housekeeping/README.rst [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+Housekeepers are maintaining and servicing the guest rooms and replenishing stock as and when needed.
+
+You can manage:
+
+**Housekeeping Activities**
+
+**Complete Housekeeping Process**
+
+    In Dirty Stage Housekeeping Process
+    .. image:: https://raw.githubusercontent.com/OCA/vertical-hotel/11.0/hotel_housekeeping/static/description/HHK1.png
```
Comment (JayVora-SerpentCS) [inline hotel_housekeeping/README.rst:38]: Path is still 11.0

File: hotel_housekeeping/models/hotel_housekeeping.py [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+        required=True,
+        states={"done": [("readonly", True)]},
+        index=True,
+    )
+    activity_line_ids = fields.One2many(
+        "hotel.housekeeping.activities",
+        "housekeeping_id",
+        "Activities",
+        states={"done": [("readonly", True)]},
+        help="Detail of housekeeping \
```
Comment (JayVora-SerpentCS) [inline hotel_housekeeping/models/hotel_housekeeping.py:42]: please do pep-8 right.
Please check and do with https://github.com/OCA/maintainer-tools#developers
Thanks,

Comment (JayVora-SerpentCS) [reply hotel_housekeeping/models/hotel_housekeeping.py:42]: Do this for all other PRs. thank you for the good work.

File: hotel_housekeeping/models/hotel_housekeeping.py [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+    )
+
+    def action_set_to_dirty(self):
+        """
+        This method is used to change the state
+        to dirty of the hotel housekeeping
+        ---------------------------------------
+        @param self: object pointer
+        """
+        self.write({"state": "dirty", "quality": False})
```
Comment (JayVora-SerpentCS) [inline hotel_housekeeping/models/hotel_housekeeping.py:91]: 'Quality' selection does not not need to pass False if nothing is set. Did you check its impact ?

**Review by JayVora-SerpentCS — CHANGES_REQUESTED** [review]

**Pull request merged**